pxtblocks/fields/field_gridpicker.ts

```diff
@@ -374,10 +394,18 @@ namespace pxtblockly {
                 menuItem.setCheckable(true);
                 menuItem.setChecked(value == this.value_);
                 menu.addChild(menuItem, true);
+                if (i == 0) {
+                    this.firstItem_ = menuItem;
+                }
             }
 
             // Listen for mouse/keyboard events.
-            goog.events.listen(menu, goog.ui.Component.EventType.ACTION, callback);
+            goog.events.listen(menu, goog.ui.Component.EventType.ACTION, (e: any) => {
```

Comment:
What happens if you scroll the inventory picker with touch? Does it select the item when you stop the scroll? That's the current behavior (and is a bug), I'm wondering if `EventType.Action` fixes this